test/invisible_ink_test.rb

```diff
@@ -3,22 +3,23 @@
 require "mocha/minitest"
 require "./lib/night_writer"
 require "./lib/invisible_ink"
+require "./lib/braille"
+
 
 class InvisibleInkTest < Minitest::Test
   def setup
     @message_file = "message.txt"
     @new_file = "braille.txt"
     @invisible_ink = InvisibleInk.new(@message_file, @new_file)
     @invisible_ink.stubs(:read_file).returns("Hi")
-
   end
 
   def test_it_exists
     assert_instance_of InvisibleInk, @invisible_ink
   end
 
   def test_it_has_attributes
-    assert_equal @message_file, @invisible_ink.message
+    assert_equal @message_file, @invisible_ink.message_file
```

Comment:
Renamed this so that it was clear that this instance variable was holding a txt file path instead of the actual message